web/components/nav/sidebar.tsx

```diff
@@ -46,8 +46,16 @@ const signedOutNavigation = [
 const signedOutMobileNavigation = [
   { name: 'Charity', href: '/charity', icon: HeartIcon },
   { name: 'Leaderboards', href: '/leaderboards', icon: CakeIcon },
-  { name: 'Discord', href: 'https://discord.gg/eHQBNBqXuh', icon: IconFromUrl("/discord-logo.svg") },
-  { name: 'Twitter', href: 'https://twitter.com/ManifoldMarkets', icon: IconFromUrl("/twitter-logo.svg") },
+  {
+    name: 'Discord',
+    href: 'https://discord.gg/eHQBNBqXuh',
+    icon: IconFromUrl('/discord-logo.svg'),
+  },
+  {
+    name: 'Twitter',
+    href: 'https://twitter.com/ManifoldMarkets',
+    icon: IconFromUrl('/twitter-logo.svg'),
+  },
```

Comment:
Prettier randomly snuck this change in, but it's what prettier wants to do, so I may as well leave it in.